vercel.json

```diff
@@ -1,5 +1,11 @@
 {
   "rewrites":  [
     {"source": "/(.*)", "destination": "/"}
+  ],
```

Comment:
- rename vercel.json.example
- by default when you fork the project, you don't have any cron active
- add documentation in readme to tell people to duplicate it and setup the key
- like that the key is fully private